distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestSecurity.java

```diff
@@ -230,6 +230,28 @@ public void testBasicRestAccess() throws Exception {
     }
 
     @Test
+    public void testSamlFederatedAuth() throws Exception {
+        String recordId = TestCatalog.ingest(Library.getSimpleGeoJson(), "application/json");
+        configureRestForBasic();
+
+        // Creating a new OpenSearch source with no username/password.
+        // When an OpenSearch source attempts to authenticate without a username/password it will
+        // use the subject in the request to create a SAML authentication token
+        OpenSearchSourceProperties openSearchProperties = new OpenSearchSourceProperties(
+                OPENSEARCH_SOURCE_ID);
+        createManagedService(OpenSearchSourceProperties.FACTORY_PID, openSearchProperties);
```

Comment:
testBasicFederatedAuth also creates an OpenSearch source.  This test should use a different source ID.